### PR TITLE
Env

### PR DESCRIPTION
**Pull request by AlexKempen**
Adds env variables and a .env file to the project.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Developer level configuration settings
+
+# Set to "production" for a production server and enviornment, or "development" for local server
```
Comment (democat3457) [inline .env:3]:
```suggestion
# Set to "production" for a production server and environment, or "development" for local server
```
fix typo

File: .env [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+NODE_ENV="development"
```
Comment (democat3457) [inline .env:4]: maybe shorten this to prod/dev?

File: src/server/api/api.ts [hunk 3]
```diff
@@ -67,10 +68,11 @@ export const websocketHandler: WebsocketRequestHandler = (ws, req) => {
 export const apiRouter = Router();
 
 apiRouter.get("/get-ids", (_, res) => {
-    return res.send({
-        // ids: ["10", "11"],
-        ids: tcpServer.getConnectedIds(),
-    });
+    const ids = tcpServer.getConnectedIds();
+    if (IS_DEVELOPMENT) {
+        ids.push("dummy-id-1", "dummy-id-2");
```
Comment (democat3457) [inline src/server/api/api.ts:73]: do non-number IDs work with the rest of the system?

File: src/server/utils/env.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { config } from "dotenv";
+
+config();
+console.log(process.env.NODE_ENV);
```
Comment (democat3457) [inline src/server/utils/env.ts:4]: Add a prefix to this so we know what the print entails in the console

File: .env (hunk 2 above)
Comment (AlexKempen) [reply .env]: No, these are de facto standards

File: src/server/utils/env.ts (hunk 4 above)
Comment (AlexKempen) [reply src/server/utils/env.ts]: Whoops, errant console.log

File: src/server/api/api.ts (hunk 3 above)
Comment (AlexKempen) [reply src/server/api/api.ts:73]: I would certainly hope so

File: .env (hunk 1 above)
Comment (AlexKempen) [reply .env]: Spell check wasn't enabled for .env, fixed now

**Review by democat3457 — APPROVED** [review]
